resolver/src/test/java/io/netty/resolver/DefaultHostsFileEntriesResolverTest.java

```diff
@@ -15,24 +15,42 @@
  */
 package io.netty.resolver;
 
+import com.google.common.collect.Maps;
 import io.netty.util.NetUtil;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
 
 import java.net.Inet4Address;
 import java.net.Inet6Address;
 import java.net.InetAddress;
+import java.nio.charset.Charset;
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.instanceOf;
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.any;
 
 public class DefaultHostsFileEntriesResolverTest {
+    private static final Map<String, List<InetAddress>> LOCALHOST_V4_ADDRESSES =
+            Collections.singletonMap("localhost", Collections.<InetAddress>singletonList(NetUtil.LOCALHOST4));
+    private static final Map<String, List<InetAddress>> LOCALHOST_V6_ADDRESSES =
+            Collections.singletonMap("localhost", Collections.<InetAddress>singletonList(NetUtil.LOCALHOST6));
+
+    @BeforeAll
+    public static void beforeAll() {
+        System.setProperty("io.netty.hostsFileRefreshInterval", String.valueOf(TimeUnit.MINUTES.toNanos(1)));
```

Comment:
See above... if we add a constructor that takes the interval we can directly construct the `DefaultHostsFileEntriesResolver` in tests and set the correct interval.

Reply:
replaced this `beforeAll` with `refreshInterval` in constructor params